### PR TITLE
2 packages from sapristi/easy_logging at 0.4

### PR DESCRIPTION
**Pull request by sapristi**
This pull-request concerns:
-`easy_logging.0.4`: Module to easily log messages
-`easy_logging_yojson.0.4`: Configuration loader for easy_logging with yojson backend

---
* Homepage: https://sapristi.github.io/easy_logging/
* Source repo: git+https://github.com/sapristi/easy_logging.git
* Bug tracker: https://github.com/sapristi/easy_logging/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

**Comment by kit-ty-kate** [comment]
The tests fail with:
```
+ /Users/travis/.opam/opam-init/hooks/sandbox.sh "build" "dune" "runtest" "-p" "easy_logging" (CWD=/Users/travis/.opam/ocaml-system/.opam-switch/build/easy_logging.0.4)
- File "conf_loader/yojson/example/dune", line 5, characters 7-26:
- 5 |   (pps ppx_deriving_yojson))
-            ^^^^^^^^^^^^^^^^^^^
- Error: Library "ppx_deriving_yojson" not found.
- Hint: try: dune external-lib-deps --missing -p easy_logging @runtest
```

File: packages/easy_logging/easy_logging.0.4/opam [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+homepage: "https://sapristi.github.io/easy_logging/easy_logging/"
+bug-reports: "https://github.com/sapristi/easy_logging/issues"
+dev-repo: "git+https://github.com/sapristi/easy_logging.git"
+depends: [
+  "ocaml" {>= "4.04.0"}
+  "dune" {build & >= "1.0"}
+]
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name] {with-test}
```
Comment (kit-ty-kate) [inline packages/easy_logging/easy_logging.0.4/opam]: you can (not mandatory) also add `"-j" jobs` to the test command

Comment (sapristi) [reply packages/easy_logging/easy_logging.0.4/opam]: Thanks @kit-ty-kate. What's the end purpose of this parameter ? Is there any reason to limit the number of running jobs ?

Comment (mseri) [reply packages/easy_logging/easy_logging.0.4/opam]: I think that if the number of jobs is not much higher than the CPU cores you do less context switching. I thought that withouth -j dune would use only a core, but reading the docs this seems to have changed

**Comment by mseri** [comment]
Thanks. Further changes can be added later.

**Pull request merged**